src/raw.rs

```diff
@@ -156,6 +156,7 @@ extern "C" {
 ///////////////////////////////////////////////////////////////
 
 pub const FMT: *const c_char = b"v\0".as_ptr() as *const c_char;
+pub const REDISMODULE_HASH_DELETE: *const RedisModuleString = 1 as *const RedisModuleString;
```

Comment:
I tried to figure out why this was needed -- I expected `bindgen` to generate this automatically.
It turns out it doesn't because it can't parse the typecast, see https://github.com/rust-lang/rust-bindgen/issues/316.
Please add a comment here that mentions this.